.travis.yml

```diff
@@ -0,0 +1,18 @@
+---
+language: go
+sudo: false
+go:
+  - 1.7.3
+  - 1.8.1
+  - tip
+
+matrix:
+  allow_failures:
+    - go: tip
+
+install:
+  - go get -u ./...
+  - go get -u gopkg.in/stretchr/testify.v1/assert
```

Comment:
Add `-t` to `go get` to also download test dependencies. No need to separately get packages.

```sh
go get -u -t ./...
```